README.md

```diff
@@ -51,3 +51,111 @@ Install an OpenPGP implementation:
   to a package called `gnupg2`
 * Android: https://www.openkeychain.org/
 * iOS: https://privacyapp.io/
+
+## After the event
+
+You should now have a table with a set of validated fingerprints and ids. You
+can use this to now sign any of those identities that you have personally
```

Comment:
Maybe highlight the words "personally validated"

Reply:
good plan